src/mkdocs_coverage/plugin.py

```diff
@@ -26,9 +24,10 @@
 class MkDocsCoverageConfig(Config):
     """Configuration options for the plugin."""
 
-    page_name = Optional(MkType(str, default=None))
+    page_name = MkDeprecated(moved_to="page_path", option_type=MkOptional(MkType(str, default=None)))
```

Comment:
We should reuse the same message as before:


```suggestion
    page_name = MkDeprecated(
        moved_to="page_path",
        option_type=MkOptional(MkType(str, default=None)),
        message="The 'page_name' configuration option is deprecated and will be removed in a future release. "
                "Use the 'page_path' configuration option instead."
    )
```